openhands/llm/llm.py

```diff
@@ -70,11 +70,6 @@ def __init__(
         # Set up config attributes with default values to prevent AttributeError
         LLMConfig.set_missing_attributes(self.config)
 
-        self.supports_prompt_caching = (
```

Comment:
@tobitege here is the difference, I discarded both, and won't check ` vision_is_active() ` when activing prompt cache